Agents/IfcOwlConverterAgent/Dockerfile

```diff
@@ -0,0 +1,54 @@
+# First stage: build war file for main agent
+#==================================================================================================
+FROM maven:3.8.6-eclipse-temurin-11-focal as builder
+
+# Copy Maven settings templates and credentials
+COPY /credentials /root/credentials
+COPY /.m2 /root/.m2
+
+# Populate settings templates with credentials, repo name
+WORKDIR /root/.m2
+# (Note that | rather than / is used as the sed delimiter, since encrypted passwords can contain the former, but not the latter
```

Comment:
"latter, but not the former"

Reply:
Made the changes, and did so for the other PRs.